shared/js/project.js

```diff
@@ -190,7 +200,17 @@ module.exports = {
 
     current = data;
     if (isNewChannel) {
-      location.href = current.level + '#' + current.id + '/edit';
+      // We have a new channel, meaning either we had no channel before, or
+      // we've changed channels.
+
+      if (location.hash || !window.history.pushState) {
+        // We're using a hash route or don't support replace state. Use our hash
+        // based route to ensure we don't have a page load.
+        location.href = current.level + '#' + current.id + '/edit';
+      } else {
+        window.history.pushState(null, document.title,
```

Comment:
Hmm. I used pushState here vs. replaceState elsewhere thinking that DID get us back behavior. May need to investigate further.